reark/src/main/java/io/reark/reark/data/stores/StoreGetInterface.java

```diff
@@ -0,0 +1,10 @@
+package io.reark.reark.data.stores;
+
+import rx.Observable;
+
+/**
+ * Created by ttuo on 27/06/16.
+ */
+public interface StoreGetInterface<T, U> {
+    Observable<U> getStream(T id);
```

Comment:
This getStream() should probably still have the behaviour of returning the first value? Hard to think of a case where you'd want the stream but not the cache value.